src/njoy21/input/ACER/Card7/test/Card7.test.cpp

```diff
@@ -0,0 +1,57 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Verifying ACER Card7 input", 
+         "[ACER], [Card7]" ){
+  GIVEN( "all three entries" ){
+    THEN( "the appropriate values are returned" ){
+      iRecordStream<char> issCard7( std::istringstream( "-2.0 1E3 1/" ) );
+      ACER::Card7 card7( issCard7 );
+
+      REQUIRE(-2.00 == card7.thin1.value );
+      REQUIRE( 1E3 == card7.thin2.value );
+      REQUIRE( 1   == card7.thin3.value );
+    }
+  }
+  GIVEN( "only two entries" ){
+    THEN( "the appropriate values are returned" ){
+      iRecordStream<char> issCard7( std::istringstream( "-2.0 1E3 /" ) );
+      ACER::Card7 card7( issCard7 );
+
+      REQUIRE(-2.00 == card7.thin1.value );
+      REQUIRE( 1E3 == card7.thin2.value );
+      REQUIRE( 0   == card7.thin3.value );
+    }
+  }
+  GIVEN( "only one entry" ){
+    THEN( "the appropriate values are returned" ){
+      iRecordStream<char> issCard7( std::istringstream( "-2.0 /" ) );
```

Comment:
doesn't make sense. thin values between 2.0 (lower bound) and 0.0(upper bound)? You provided an argument, so you probably want _some_ thinning